spec/save_inventory/init_data_helper.rb

```diff
@@ -1,90 +1,93 @@
+require_relative "../persister/test_base_persister"
+require_relative "../persister/test_builder/cloud_manager"
+require_relative "../persister/test_builder/network_manager"
+
 module InitDataHelper
```

Comment:
I think all of this should be just rewritten to the persister format? Now it's like half way there?

Reply:
What do you mean exactly? all definitions in this file are rewritten to persister format